### PR TITLE
Flight results integration

### PR DESCRIPTION
**Pull request by Chris-DJ-Kim**
Flight results for both incoming and outbound flights are obtained from flight scanner api. Details for both routes (if available) are returned. Only rudimentary styling on returned details.

### CODE REVIEW

File: client/src/components/Search.js
Comment (vi-nguyen-12) [inline client/src/components/Search.js:59]: I just pushed an axios file for FE to call Api easier. would you have a look and apply it here. it's like:
 const handleSubmit=async ()={
const response= await skyscannerApi.getQuotes(data);
...
}
It's just my opinion, what do you think?

Comment (Chris-DJ-Kim) [reply client/src/components/Search.js:59]: I took a look and looks good! I'll need to wait until your files are merged as there are likely edits I need to make to the front end components to accommodate. Once you merge I will update from the dev and dig into it!

File: client/src/components/flight-details-by-direction.component.js
Comment (jusa798) [inline client/src/components/flight-details-by-direction.component.js]: I like that you thought about the hover. You could also probably add the colors to the theme so we can keep the styling consistent. Nice work on the timeline!

File: client/src/components/flight-details-by-direction.component.js
Comment (sina-jamshidi) [inline client/src/components/flight-details-by-direction.component.js:27]: nice, easy to read

File: client/src/components/flight-details.component.js
Comment (sina-jamshidi) [inline client/src/components/flight-details.component.js]: Can't we combine this into getFlightDetails and just have it return two objects: `outgoingDetails` and `inboundDetails`?

File: client/src/utils/skyscanner.js
Comment (sina-jamshidi) [inline client/src/utils/skyscanner.js:20]: we can leave this for now since there will be autocomplete as a separate task later

**Review by sina-jamshidi — APPROVED** [review]
Looks good!
Chloe's PR will be merged soon, so you can update this.

File: client/src/components/flight-details.component.js
Comment (Chris-DJ-Kim) [reply client/src/components/flight-details.component.js]: Yes, just combined it! The repetitive part is now handled within the getFlightDetails function.

Comment (Chris-DJ-Kim) [reply client/src/components/flight-details.component.js]: Actually, getFlightDetails was repetitive to begin with so not much changed on that part

**Comment by Chris-DJ-Kim** [comment]
If anyone wonders why there are two reverts in the commit history...there was a misclick.

**Pull request merged**
